dev-libs/libmspack/libmspack-0.5~alpha.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="A portable library for Microsoft compression formats"
+DESCRIPTION="The following file formats are supported:
+* SZDD files, which use LZSS compression
+* KWAJ files, which use LZSS, LZSS+Huffman or deflate compression
+* .HLP (MS Help) files, which use LZSS compression
+* .CAB (MS Cabinet) files, which use deflate, LZX or Quantum compression
+* .CHM (HTML Help) files, which use LZX compression
+* .LIT (MS EBook) files, which use LZX compression and DES encryption
+* .LZX (Exchange Offline Addressbook) files, which use LZX compression
+
+To determine the capabilities of the library, and the binary compatibility \
+version of any particular compressor or decompressor, use the \
+mspack_version() function. The UNIX library interface version is \
+defined as the highest-versioned library component."
+HOMEPAGE="https://www.cabextract.org.uk/libmspack/"
+COPYRIGHT="2000-2015 Stuart Caie"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/libmspack-${portVersion/\~/}.tar.gz"
+CHECKSUM_SHA256="8967f275525f5067b364cee43b73e44d0433668c39f9376dfff19f653d1c8110"
+SOURCE_DIR="libmspack-${portVersion/\~/}"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libmspack$secondaryArchSuffix = $portVersion
+	lib:libmspack$secondaryArchSuffix = 0.1.0 compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libmspack${secondaryArchSuffix}_devel = $portVersion
+	devel:libmspack$secondaryArchSuffix = 0.1.0 compat >= 0
+	"
+REQUIRES_devel="
+	libmspack$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autoheader
+	cmd:automake
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	runConfigure ./configure
+	if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
+		make $jobArgs AM_CFLAGS=-Wall
+	else
+		make $jobArgs
+	fi
+}
+
+INSTALL()
+{
+	make install
+
+	if [ -z "$secondaryArchSuffix" ]; then
+		install -d $docDir $developDocDir
+		install -t $docDir AUTHORS README TODO
```

Comment:
Please replace
```
		install -t $docDir AUTHORS README TODO
```
by
```
		install -t $docDir AUTHORS README
```
(The TODO file is not that interesting, after all, and we would need to have one more test at the end of INSTALL.)